### ISSUE TITLE
unable to login after upgrading to awx-operator v2.2.0

### ISSUE THREAD

**Issue by dwsorge** [ansible/tower-operator#1619]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx-operator/issues) for duplicates.
- [X] I understand that the AWX Operator is open source software provided for free and that I might not receive a timely response.

### Bug Summary

After attempting to upgrade awx-operator from v2.1.0 to v2.2.0 I am unable to login to the GUI.
Both local admin password and LDAP logins fail. 

Previous upgrades between minor/major versions have worked fine.
Upgrading between these versions is where I begin to see the problem.

### AWX Operator version

v2.2.0

### AWX version

v22.3.0

### Kubernetes platform

kubernetes

### Kubernetes/Platform version

v1.24.13

### Modifications

yes

### Steps to reproduce

I am upgrading using the following command

```
[master-node]# cd /awx-operator-2.2.0/awx-operator/
[master-node]# helm upgrade -n awx-operator awx-operator -f values.yaml .
Release "awx-operator" has been upgraded. Happy Helming!
NAME: awx-operator
LAST DEPLOYED: Fri Oct 27 15:13:16 2023
NAMESPACE: awx-operator
STATUS: deployed
REVISION: 34
TEST SUITE: None
NOTES:
AWX Operator installed with Helm Chart version 2.2.0
```

pods appear to be healthy

```
[master-node]# kgpa
NAMESPACE        NAME                                                                READY   STATUS      RESTARTS         AGE
awx-operator     awx-demo-postgres-13-0                                              1/1     Running     0                16d
awx-operator     awx-demo-task-5798478475-nmkm7                                      4/4     Running     0                13m
awx-operator     awx-demo-web-86f874b58d-5p7fl                                       3/3     Running     0                13m
awx-operator     awx-operator-controller-manager-7d54447789-l2fcr                    2/2     Running     0                14m
```

### Expected results

I expected healthy pods in the awx-operator namespace and normal GUI functionality.

### Actual results

Pods show as healthy but login attempts (local and LDAP) fail. 

### Additional information

In our environment I am pulling kube-rbac-proxy:v0.13.0 from an internal repository.
I modified templates/deployment-awx-operator-controller-manager.yaml to point to the internal repo.

### Operator Logs

[master-node]# k logs -n awx-operator deployments/awx-demo-web awx-demo-web

<IP_REDACTED> - - [27/Oct/2023:15:23:13 +0000] "GET /api/login/ HTTP/1.1" 200 5710 "https://<AWX_URL>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36" "-"
[pid: 34|app: 0|req: 2/7] <IP_REDACTED> () {60 vars in 5006 bytes} [Fri Oct 27 15:23:12 2023] GET /api/login/ => generated 5710 bytes in 85 msecs (HTTP/1.1 200) 10 headers in 460 bytes (1 switches on core 0)
2023-10-27 15:23:13,125 WARNING  [b446716e458a4178b31abe44e5a97c30] django.security.csrf Forbidden (Origin checking failed - https://<AWX_URL> does not match any trusted origins.): /api/login/
<IP_REDACTED> - - [27/Oct/2023:15:23:13 +0000] "POST /api/login/ HTTP/1.1" 403 1019 "https://<AWX_URL>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36" "-"
[pid: 36|app: 0|req: 3/8] <IP_REDACTED> () {66 vars in 5175 bytes} [Fri Oct 27 15:23:13 2023] POST /api/login/ => generated 1019 bytes in 64 msecs (HTTP/1.1 403) 7 headers in 276 bytes (1 switches on core 0)
<IP_REDACTED>- - [27/Oct/2023:15:25:25 +0000] "GET /api/login/ HTTP/1.1" 200 5710 "https://<AWX_URL>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36" "-"
[pid: 34|app: 0|req: 3/9] <IP_REDACTED> () {60 vars in 5005 bytes} [Fri Oct 27 15:25:25 2023] GET /api/login/ => generated 5710 bytes in 59 msecs (HTTP/1.1 200) 10 headers in 460 bytes (1 switches on core 0)
2023-10-27 15:25:26,382 WARNING  [ac78fce847994f7a8ed60ad1c745907f] django.security.csrf Forbidden (Origin checking failed - https://<AWX_URL> does not match any trusted origins.): /api/login/
<IP_REDACTED> - - [27/Oct/2023:15:25:26 +0000] "POST /api/login/ HTTP/1.1" 403 1019 "https://<AWX_URL>/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36" "-"
[pid: 35|app: 0|req: 1/10] <IP_REDACTED> () {66 vars in 5174 bytes} [Fri Oct 27 15:25:25 2023] POST /api/login/ => generated 1019 bytes in 494 msecs (HTTP/1.1 403) 7 headers in 276 bytes (1 switches on core 0)

**Comment by fredddy255f** [comment]
hello, I have the same problem ...

edit: in my case (fresh install in minikube), i have resolved my issue with https://github.com/ansible/awx/issues/14024 .

**Comment by dwsorge** [comment]
Looks like my issue was resolved with ansible/awx#14024 as well.
